### ISSUE TITLE
portal portal still in command list

### ISSUE THREAD

**Issue by Managarmr719** [sekwah41/Advanced-Portals#48]
The command /portal portal still appears in the command list when typing /portal help even tough it doesn't work and the changelog for the latest version mentions it being removed ("[Removed portal portal as it did nothing.](https://www.spigotmc.org/resources/advanced-portals.14356/update?update=108243)").

EDIT: I'm using version 0.0.21.

**Comment by sekwah41** [comment]
I it works in one of the older though support has been removed for it. I will add a new build which adds a placeholder for you to place.

**Comment by sekwah41** [comment]
This is listed in another issue so i will close it for now though i will post on here once it is done :)
